src/subscription_manager/repofile.py

```diff
@@ -461,19 +461,28 @@ def sections(self):
 
         def fix_content(self, content):
             # Luckily apt ignores all Fields it does not recognize
-            baseurl = content["baseurl"]
+            parsed_url = urlparse(unquote(content["baseurl"]))
```

Comment:
is `unquote()` still needed? see the discussion on this in #3223

Reply:
At least the Candlepin version available for Katello 4.13 (the latest supported release), currently Candlepin 4.4.14 is still affected. I have spent some time looking, but was not able to find any new information when or if Candlepin intends to go back to the old behavior. Given nothing has happened over several months, I can only assume it is not a major priority. The good news is that using `unquote` will work equally well for quoted and unquoted URLs. As a result I lean towards leaving it in, at least until there is any new information from Candlepin.

Reply:
It looks like that while some encoding issues in Candlepin were [fixed](https://github.com/candlepin/candlepin/pull/4783), this does not appear to have changed that the baseurl used as part of these changes has its path part encoded. Not sure if this is intentional or a bug on Candlepin's part. The current state of these changes will work either way.